bin/addons-linter

```diff
@@ -4,7 +4,13 @@ var path = require('path');
 
 var absoluteAppRoot = path.resolve(path.join(__dirname, '..'));
 global.appRoot = path.relative(process.cwd(), absoluteAppRoot);
-
-require('../dist/addons-linter')
-  .createInstance({runAsBinary: true})
-  .run();
+try{
+    require('../dist/addons-linter')
+    .createInstance({runAsBinary: true})
+    .run();
+}
+catch (e) {
+    if (e.code == 'MODULE_NOT_FOUND') {
+       console.log('You did not build addons-linter yet. Please run \'npm install\' and \'yarn build\' or see README.md for more information');
```

Comment:
As mentioned please use just one command for both things. Looks like the README leans toward `yarn`, so this should read `"Please run `yarn` and `yarn build` or see README.md for more information"`.

Note also the use of double-quotes; they're fine to use here because it saves you having the escape the `'` character.